src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -0,0 +1,59 @@
+<div class="pf-c-input-group">
+  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+  {{#> form-controls controlType="textarea" name="textarea" id="textarea1" aria-label="textarea example"}}
+{{/form-controls}}
+    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+</div> 
+<br>
+<br>
```

Comment:
I imagine @andresgalante would say, scope workspace styling for spacing instead of adding `<br>`s. Not a blocker.

Reply:
ya I've used this approach in alerts as well. Do you think we should break it out into separate examples?

Reply:
No, I think it's a good approach